pkg/server/auth/server.go

```diff
@@ -109,19 +112,30 @@ func NewAuthServer(ctx context.Context, cfg AuthConfig) (*AuthServer, error) {
 		if err != nil {
 			return nil, fmt.Errorf("could not create provider: %w", err)
 		}
+		// Set isOidcEnabled now we have a valid provider
+		isOidcEnabled = true
```

Comment:
Why not call `SetOidcEnabled()` here?

Reply:
because `SetOidcEnabled` is intended for testing only and frankly shouldn't exist. I'd like to minimise it's usage to test files only so if we can remove it it's as easy to remove as possible